org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/XMLTextDocumentService.java

```diff
@@ -76,7 +76,7 @@
 	private final XMLLanguageServer xmlLanguageServer;
 	private final TextDocuments documents;
 	private final LanguageModelCache<XMLDocument> xmlDocuments;
-	private final CompletionSettings sharedCompletionSettings;
+	private CompletionSettings sharedCompletionSettings;

```

Comment:
Why do you remove final?

Reply:
@angelozerr Sorry, I fixed all your recommendations before but I pushed the wrong version.

`Why do you remove final?`

I did this since Completion Settings can change because autoCloseTags is a user preference and autoCloseTags is in CompletionSettings